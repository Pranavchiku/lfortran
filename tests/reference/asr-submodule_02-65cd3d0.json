{
    "basename": "asr-submodule_02-65cd3d0",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/submodule_02.f90",
    "infile_hash": "eb5651198a139f1a3e4bda3dd45dc20f0f6ceb8f961eff3d2cff2342",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-submodule_02-65cd3d0.stdout",
    "stdout_hash": "bd462a6b0da8f72cf12893a0f07192f72b2e79360aa8f7871193d7a6",
    "stderr": "asr-submodule_02-65cd3d0.stderr",
    "stderr_hash": "66aff2a6970665e02050ada03bff2864d6f1c07aa9017e4ece83f433",
    "returncode": 0
}