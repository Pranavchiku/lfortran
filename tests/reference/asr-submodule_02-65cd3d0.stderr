warning: Implied save feature
 --> tests/../integration_tests/submodule_02.f90:6:1
  |
6 | public rkind1, int32
  | ^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/submodule_02.f90:6:1
  |
6 | public rkind1, int32
  | ^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:14:6
   |
14 |      real :: x, y
   |      ^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:14:6
   |
14 |      real :: x, y
   |      ^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:19:8
   |
19 |        type(point), intent(in) :: a, b
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:19:8
   |
19 |        type(point), intent(in) :: a, b
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:20:8
   |
20 |        real(rkind) :: distance
   |        ^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:26:8
   |
26 |        type(point), intent(in) :: a, b
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:26:8
   |
26 |        type(point), intent(in) :: a, b
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:27:8
   |
27 |        real(rkind), intent(out) :: distance
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:35:5
   |
35 |     type(point), intent(in) :: a, b
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:35:5
   |
35 |     type(point), intent(in) :: a, b
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:36:5
   |
36 |     real(rkind) :: distance
   |     ^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:41:5
   |
41 |     type(point), intent(in) :: a, b
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:41:5
   |
41 |     type(point), intent(in) :: a, b
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/submodule_02.f90:42:5
   |
42 |     real(rkind), intent(out) :: distance
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
